k6ext/panic.go

```diff
@@ -33,21 +33,23 @@ func Panic(ctx context.Context, format string, a ...any) {
 	}
 	defer k6common.Throw(rt, fmt.Errorf(format, a...))
 
-	pid := GetProcessID(ctx)
-	if pid == 0 {
-		// this should never happen unless a programmer error
-		panic("no browser process ID in context")
-	}
-	p, err := os.FindProcess(pid)
-	if err != nil {
-		// optimistically return and don't kill the process
+	pidder, ok := GetVU(ctx).(interface {
```

Comment:
I was trying to improve this hack, but as you said @inancgumus I think the best way to fix it is in the long run, by moving panics only to the mapping layer were we would have access directly to the `moduleVU` which maintains the `pid` list and therefore will be able to avoid this casting.

Reply:
Exactly! Don't worry about it. I'll put a `TODO` comment there so we won't forget.